From e1af12a4163200e119263fb16eb2ab2da0e438a7 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 13 Sep 2013 11:02:32 +0200
Subject: [GNOME-ABRT PATCH 3/3] Do not change old selection upon reloading

Closes #29

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/gnome_abrt/views.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gnome_abrt/views.py b/src/gnome_abrt/views.py
index d85affc..2abacc3 100644
--- a/src/gnome_abrt/views.py
+++ b/src/gnome_abrt/views.py
@@ -485,7 +485,6 @@ class OopsWindow(Gtk.ApplicationWindow):
         old_selection = self._get_selected(self._builder.tvs_problems)
 
         self._reloading = True
-        old_selection = None
         try:
             self._builder.ls_problems.clear()
 
-- 
1.8.3.1

